### DISCUSSION TITLE
how-to-shift-copyright-page-to-the-end-?

### DISCUSSION THREAD

**Discussion by vishu-byte** [joaomlourenco/novathesis#378] [Q&A] [answered]
I need to shift the copyright page to the end and also make some changes to it. Can anyone tell me how should I go about doing this?

Thanks

Vishu

**Comment by joaomlourenco** [comment, marked as answer]
If the "School" has a copyright text, it will be used.  You can find examples in:
```
./NOVAthesisFiles/copyright.ldf
./NOVAthesisFiles/Schools/uminho/copyright.ldf
./NOVAthesisFiles/Schools/nova/copyright.ldf
```

Concetrning the “moving to the end”, please clarify *precisely* where do you want the copyright text to go.
